debian/postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html
+# Boradly speaking, the `postinst` is called after a package is unpacked.
+
+set -e
+
+/opt/one_compiler/bin/one-prepare-venv
```

Comment:
If user can change installation path to somewhere else, is `/opt/one_compiler/bin/one-prepare-venv` available?

Reply:
@seanshpark Umm I thought user can't change the installation path. Is it possible?

Reply:
> Is it possible?

I don't know either...
But with some searching, it seems not possible :)